session/manager_http.go

```diff
@@ -320,3 +323,28 @@ func (s *ManagerHTTP) SessionAddAuthenticationMethods(ctx context.Context, sid u
 	sess.SetAuthenticatorAssuranceLevel()
 	return s.r.SessionPersister().UpsertSession(ctx, sess)
 }
+
+func (s *ManagerHTTP) MaybeRedirectAPICodeFlow(w http.ResponseWriter, r *http.Request, f flow.Flow, sessionID uuid.UUID) (handled bool, err error) {
+	ctx, span := s.r.Tracer(r.Context()).Tracer().Start(r.Context(), "sessions.ManagerHTTP.MaybeRedirectAPICodeFlow")
+	defer otelx.End(span, &err)
+
+	if code, ok, _ := s.r.SessionTokenExchangePersister().CodeForFlow(ctx, f.GetID()); ok {
+		returnTo := s.r.Config().SelfServiceBrowserDefaultReturnTo(ctx)
```

Comment:
No need to change, just a remark. It's often better to read when the short code path is indented and the long one not:

```
if true {
  // short
}
// very
// long
// execution
```

Reply:
You're right! I extracted it previously, but it's nicer when the guard clause comes first.